### DISCUSSION TITLE
Usage statistics for (Helm) repository

### DISCUSSION THREAD

**Discussion by DTMad** [artifacthub/hub#1720] [Ideas]
Hey,

since I/we maintain multiple different packages I'd love to see some usage statistics for them in ArtifactHub.
Is there a way to see how often the Helm chart has been clicked or how often the "Install" button got clicked?

Thanks,
Marco

**Comment by tegioz** [comment]
Hi @DTMad 👋 

This is not possible at the moment as this information is not being tracked yet. But it's something interesting and we'll definitely consider it, thanks 👍

**Comment by tegioz** [comment]
Tomorrow we'll start [tracking packages views](https://github.com/artifacthub/hub/pull/1728). We'll collect data for a few days and then we'll add a new view to display it in the UI 🙂

**Reply by DTMad** [reply]
Awesome, thanks!

**Reply by tegioz** [reply]
https://twitter.com/cncfartifacthub/status/1469390012630409221?s=20 🙂
